lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnJobSubmissionClient.java

```diff
@@ -43,8 +44,8 @@
 import org.apache.reef.tang.exceptions.InjectionException;
 import org.apache.reef.tang.formats.ConfigurationSerializer;
 import org.apache.reef.util.JARFileMaker;
-
 import javax.inject.Inject;
+import java.io.*;
```

Comment:
If we're importing `io.*` we can remove subsequent imports of individual classes. But from our code it looks like we're trying not to import * but rather list individual classes we need.